linux/keyman-config/km-config

```diff
@@ -16,7 +16,10 @@ if __name__ == '__main__':
     parser.add_argument('--version', action='version', version='%(prog)s version ' + versionstring)
     parser.add_argument('-v', '--verbose', action='store_true', help='verbose logging')
     parser.add_argument('-vv', '--veryverbose', action='store_true', help='very verbose logging')
-    parser.add_argument('-i', '--install', action='store', help='download and/or install .kmp package')
+    parser.add_argument('-i', '--install', action='store', help='download and/or install .kmp ' +
+                        'package. INSTALL can either be a downloaded .kmp file, a file:// URL ' +
+                        'pointing to a .kmp file, or a keyman:// URL, possibly with a ' +
+                        'bcp47=<language> specified.')
```

Comment:
Are the `-h --help` args already added?

From 
```
opts="-h --help -v --verbose -vv --veryverbose --version -i --install"
```

Reply:
`-h` and `--help` are built-in arguments that get added implicitly.